frontend/src/AccountCreation/SignUpProfessor.tsx

```diff
@@ -24,54 +26,101 @@ const SignUpProfessor = () => {
         {error && <Alert variant={"danger"}>Failed to login: {error}</Alert>}
         <br />
         <Form onSubmit={submit}>
-          <Form.Group className="mb-3" controlId="formCourseCode">
-            <Form.Label>Your Course Code:</Form.Label>
+          <Stack direction="horizontal" className="signUpFormText">
+            <Form.Group className="mb-3" controlId="formCourseCode">
+              <Form.Label>Your Course Code:</Form.Label>
```

Comment:
Just a thought- maybe it's better to not show the course code until it's been generated? 
For example, if the initial state of courseCode could be "" (empty string), and then we can add a ternary operator like 

`courseCode ? <courseCode code> : null`

Reply:
Done. I added a spinner when the value of the course code has not been loaded.
<img width="881" alt="Screen Shot 2022-02-22 at 11 56 04 AM" src="https://user-images.githubusercontent.com/44539346/155209127-207f1646-66ef-4bdb-bc6e-8f9b1e402af1.png">